repo/packages/H/hdfs/0/command.json

```diff
@@ -1,5 +1,5 @@
 {
   "pip": [
-    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.0#dcos-hdfs=0.1.0"
+    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.1#dcos-hdfs=0.1.1"
```

Comment:
I'm still not sure how we would send a local configuration file without encoding it.